blockchain/blockchain-wallet-v4-frontend
# Debt/bitcoinjs

## Upgrade to bitcoinjs-lib `5.1.6`
Upgrades to core and frontend in accordance with the library changes.

## Testing Steps
1. Create new wallet. Check that all elements of the kv store are successful kvStore->*. Check that the wallet has been created in wallet walletPath->wallet->hd_wallets[0].
2. Import legacy address. Check walletPath->wallet->addresses.
3. Add new wallet. Check walletPath->wallet->hd_wallets[1].
4. Connect lockbox. 
5. Sends for btc, bch, stellar, eth, and paxos.
6. Sign message in wallets/addresses settings.
7. Show private key in wallets/addresses settings.
